src/ucp/core/ucp_ep.c

```diff
@@ -2445,8 +2446,9 @@ void ucp_ep_do_keepalive(ucp_ep_h ep, ucp_lane_map_t *lane_map)
          * in connect-to-iface mode then use UCP/AM keepalive */
         if (ucp_ep_is_am_keepalive(ep, lane)) {
             rsc_index = ucp_ep_get_rsc_index(ep, lane);
+            UCS_BITMAP_SET(tl_bitmap, rsc_index);
             status    = ucp_wireup_msg_send(ep, UCP_WIREUP_MSG_EP_CHECK,
-                                            UCS_BIT(rsc_index), NULL);
+                                            &tl_bitmap, NULL);
```

Comment:
need to reset the bitmap every time because now it is reused from prev loop iteraiton